io.github.Archeb.opentrace.metainfo.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.Archeb.opentrace</id>
+  <name>OpenTrace</name>
+  <summary>A cross-platform GUI wrapper for NextTrace</summary>
+  <summary xml:lang="zh-Hans">NextTrace 的跨平台 GUI 界面</summary>
+  <developer_name>Archeb</developer_name>
+  <launchable type="desktop-id">io.github.Archeb.opentrace.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://github.com/Archeb/opentrace#GPL-3.0-1-ov-file</project_license>
+  <description>
+    <p>OpenTrace 是 NextTrace 的跨平台 GUI 界面，带来您熟悉但更强大的用户体验。</p>
```

Comment:
description should be in English. You can add a translation if you want.

Reply:
Yes it contains both English and Chinese version. I'll remove the Chinese part.